config.yml

```diff
@@ -61,12 +63,63 @@ processors:
       - key: aws.log.stream.arns
         action: delete
 
-  resource/rename:
```

Comment:
When we talked about this I somehow missed that resource/rename was actually used in the metrics pipeline, can we leave it in since it's used?